spring-cloud-gcp-security-iap/src/main/java/org/springframework/cloud/gcp/security/iap/AppEngineAudienceProvider.java

```diff
@@ -48,7 +48,7 @@ public AppEngineAudienceProvider(GcpProjectIdProvider projectIdProvider) {
 	@Override
 	public String getAudience() {
 		Project project = this.resourceManager.get(this.projectIdProvider.getProjectId());
-		Assert.notNull(project, "Project expected not to be null.");
+		Assert.notNull(project, "Project expected not to be null. Is Cloud Resource Manager API enabled?");
```

Comment:
Maybe link to the view in Cloud Console that lets you enable the API?

Reply:
Added it in code. Messages like these are very common in gcloud CLI, but don't appear in client library code, for example. Still, it will be a common issue, since Resource Manager API is not enabled by default on new projects, so it's worth being more verbose to save people time.